[BUG] The section about development in docker in README.md has incorrect commands

#### Bug Report Checklist

- [ ] Have you provided a full/minimal spec to reproduce the issue?
- [ ] Have you validated the input using an OpenAPI validator?
- [ ] Have you [tested with the latest master](https://github.com/OpenAPITools/openapi-generator/wiki/FAQ#how-to-test-with-the-latest-master-of-openapi-generator) to confirm the issue still exists?
- [x] Have you searched for related issues/PRs?
- [ ] What's the actual output vs expected output?
- [ ] [Optional] Sponsorship to speed up the bug fix or feature request ([example](https://github.com/OpenAPITools/openapi-generator/issues/6178))

<!--
Please follow the issue template below for bug reports.
Also please indicate in the issue title which language/library is concerned. E.g.:  [BUG][JAVA] Bug generating foo with bar 
-->

##### Description

<!-- describe what is the question, suggestion or issue and why this is a problem for you. -->

The "[Run Docker in Vagrant](https://github.com/OpenAPITools/openapi-generator/blob/master/README.md#run-docker-in-vagrant)" section in README.md introduces the following command,

```
./run-in-docker.sh ./mvnw package
```

but it doesn't work as follows.

```
vagrant@ubuntu-bionic:/vagrant$ ./run-in-docker.sh ./mvnw package

(snip)

3263 [INFO] ------------------------------------------------------------------------
3263 [INFO] BUILD FAILURE
3264 [INFO] ------------------------------------------------------------------------
3265 [INFO] Total time:  2.081 s
3266 [INFO] Finished at: 2026-02-14T07:35:26Z
3267 [INFO] ------------------------------------------------------------------------
3316 [INFO] 0 goals, 0 executed
52540 [WARNING] Adding build scan buildFinished actions is not allowed once their invocation has started. Newly added action will be ignored.
52577 [ERROR] Unknown lifecycle phase "/var/maven/.m2". You must specify a valid lifecycle phase or a goal in the format <plugin-prefix>:<goal> or <plugin-group-id>:<plugin-artifact-id>[:<plugin-version>]:<goal>. Available lifecycle phases are: validate, initialize, generate-sources, process-sources, generate-resources, process-resources, compile, process-classes, generate-test-sources, process-test-sources, generate-test-resources, process-test-resources, test-compile, process-test-classes, test, prepare-package, package, pre-integration-test, integration-test, post-integration-test, verify, install, deploy, pre-clean, clean, post-clean, pre-site, site, post-site, site-deploy. -> [Help 1]
```

This is because [run-in-docker.sh sets `MAVEN_CONFIG` to the maven repository path](https://github.com/OpenAPITools/openapi-generator/blob/v7.19.0/run-in-docker.sh#L17),
but [mvnw interpolates its value into the command executed](https://github.com/OpenAPITools/openapi-generator/blob/v7.19.0/mvnw#L308) and it is interpreted as a maven target.

##### openapi-generator version

<!-- which version of openapi-generator are you using, is it a regression? -->

master

##### OpenAPI declaration file content or url

<!-- if it is a bug, a json or yaml that produces it.
If you post the code inline, please wrap it with
```yaml
(here your code)
```
(for YAML code) or
```json
(here your code)
```
(for JSON code), so it becomes more readable. If it is longer than about ten lines,
please create a Gist (https://gist.github.com) or upload it somewhere else and
link it here.
  -->

##### Generation Details

<!-- 
    Prefer CLI steps, including the language, libraries and various options. 
    Providing config-based settings allows for simpler testing across CLI and plugins. 
    For examples, see https://github.com/OpenAPITools/openapi-generator/tree/master/bin/configs
-->

##### Steps to reproduce

<!-- unambiguous set of steps to reproduce the bug.-->

##### Related issues/PRs

<!-- has a similar issue/PR been reported/opened before? Please do a search in https://github.com/openapitools/openapi-generator/issues?utf8=%E2%9C%93&q=is%3Aissue%20 -->

##### Suggest a fix

<!-- if you can't fix the bug yourself, perhaps you can point to what might be
  causing the problem (line of code or commit), or simply make a suggestion -->

A simple workaround is to use `mvn` instead of `mvnw` [as described in another part of README.md](https://github.com/OpenAPITools/openapi-generator/blob/v7.19.0/README.md?plain=1#L399-L404) when using run-in-docker.sh.

--- a/README.md
+++ b/README.md
@@ -418,12 +418,12 @@ Once built, `run-in-docker.sh` will act as an executable for openapi-generator-c
 
 ##### Troubleshooting
 
-If an error like this occurs, just execute the **./mvnw clean install -U** command:
+If an error like this occurs, just execute the **mvn clean install -U** command:
 
 > org.apache.maven.lifecycle.LifecycleExecutionException: Failed to execute goal org.apache.maven.plugins:maven-surefire-plugin:2.19.1:test (default-test) on project openapi-generator: A type incompatibility occurred while executing org.apache.maven.plugins:maven-surefire-plugin:2.19.1:test: java.lang.ExceptionInInitializerError cannot be cast to java.io.IOException
 
 ```sh
-./run-in-docker.sh ./mvnw clean install -U
+./run-in-docker.sh mvn clean install -U
 ```
 
 > Failed to execute goal org.fortasoft:gradle-maven-plugin:1.0.8:invoke (default) on project openapi-generator-gradle-plugin-mvn-wrapper: org.gradle.tooling.BuildException: Could not execute build using Gradle distribution 'https://services.gradle.org/distributions/gradle-4.7-bin.zip'
@@ -438,7 +438,7 @@ cd openapi-generator
 vagrant up
 vagrant ssh
 cd /vagrant
-./run-in-docker.sh ./mvnw package
+./run-in-docker.sh mvn package
 ```
 
 ### [1.7 - NPM](#table-of-contents)

